notebooks/Neighborhood applied with kmeans.ipynb

```diff
@@ -0,0 +1,1399 @@
+{
```

Comment:
<p>Is the objective of this chart to explain the K-Means method? If yes, I think you could make a brief explanation about it in a markdown. Otherwise, I believe it would be better to leave it out of the notebook, as it's not related to our dataset.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/neo-empresarial/house_pricing_challenge/pull/41/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>